ci/scripts/start-bosh.sh

```diff
@@ -187,6 +186,7 @@ function main() {
     docker network create -d bridge --subnet=10.245.0.0/16 director_network
   fi
 
+  compilation_ops="`pwd`/ci/compilation.yml"
```

Comment:
```suggestion
  compilation_ops="$(pwd)/ci/compilation.yml"
```

alternatively you could also just use `$PWD`, or $REPO_ROOT, or determine the script's current path and put the other path relative to it in case someone starts `start-bosh.sh` manually...